src/massager.ts

```diff
@@ -23,6 +24,18 @@ export function replan(inPaths: Vec2[][], planOptions: PlanOptions): Plan {
     }
   }
 
+  // Rotate drawing around center of paper to handle plotting portrait drawings
+  // along y-axis of plotter
+  // This must be done after scaling
```

Comment:
Hm, it's surprising to me that this needs to be done after scaling. Wouldn't this make the drawing run off the edge of the page?

Further, if this was done prior to scaling, it wouldn't matter what the center of the rotation was.